src/assets/i18n/en.json5

```diff
@@ -5628,27 +5608,42 @@
 
   "access-control-remove": "Remove access condition",
 
-  "access-control-select-bitstreams-modal.title": "Select bitstreams",
+  "access-control-option-start-date-note": "Select the date from which the related access condition is applied",
+
+  "access-control-option-end-date": "Grant access until",
 
-  "access-control-select-bitstreams-modal.no-items": "No items to show.",
+  "access-control-option-end-date-note": "Select the date until which the related access condition is applied",
 
-  "access-control-select-bitstreams-modal.close": "Close",
+  "vocabulary-treeview.search.form.add": "Add",
 
-  "access-control-option-label": "Access condition type",
+  "search.filters.filter.entityType.text": "Item Type",
 
-  "access-control-option-note": "Choose an access condition to apply to selected objects.",
+  "search.filters.operator.equals.text": "Equals",
 
-  "access-control-option-start-date": "Grant access from",
+  "search.filters.operator.notequals.text": "Not Equals",
 
-  "access-control-option-start-date-note": "Select the date from which the related access condition is applied",
+  "search.filters.operator.notcontains.text": "Not Contains",
 
-  "access-control-option-end-date": "Grant access until",
+  "search.filters.operator.contains.text": "Contains",
 
-  "access-control-option-end-date-note": "Select the date until which the related access condition is applied",
```

Comment:
You've again accidently removed another 9 i18n keys here on lines 5631 to 5647.  These must **all** be restored as removing them will break other features.